src/main/java/TaskList.java

```diff
@@ -0,0 +1,76 @@
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+
+public class TaskList {
+
+
+    private ArrayList<Task> tasks;
+
+    /**
+     * Constructor for the class that stores all the tasks
+     * @param tasks
+     */
+    public TaskList(ArrayList<Task> tasks) {
+        this.tasks = tasks;
+    }
+
+
+    public TaskList() {
+    }
```

Comment:
Is this constructor for a new TaskList where the ArrayList has not been created yet? If yes, then it's best to initialize the ArrayList as empty in here!